### ISSUE TITLE
Memory efficiency

### ISSUE THREAD

**Issue by Emanuel600** [Emanuel600/Bash-Interface#2]
For now lack of memory efficiency in this program hasn't been an issue and I doubt it'll ever be. However I find it annoying when a program isn't properly optimized, and this could lead to some bigger issues later (like taking too long to finish a simple task, even as of now it takes a good 4-5 seconds sometimes just for 100ish numbers)

=> To-do <=
- [x] Change output format to binary
- [x] Change input format to binary
- [x] Change binary files on `interface` to .dat format

**Comment by Emanuel600** [comment]
### **IMPORTANT**

One file will have to be written as human readable text, so that I can see the result without having to create another executable just to translate from binary to text. I don't know how I forgot this.

**Comment by Emanuel600** [comment]
Updates:

Successfully outputted to `stdout` as binary. However the generated output seems to not be as random as I'd like, with somewhat regular `NULL`s => This seems to be a mistake on how I interpreted the output from `hexdump file`.

To read the binary without hassle, I just inputed the name of the file and had the program open it, it kind of goes against the original spirit of having the command script handle all `I/O` but it was the most practical solution from what I could tell.

Output seems to show more even numbers than odd numbers, but that might be just my own confirmation bias.
